src/utils/kungfu.ts

```diff
@@ -0,0 +1,294 @@
+
```

Comment:
Standard library file, will update manually as i add to the actual library, but would be nice to find a way to hook it to git automatically or add into node modules